pkg/server/status_test.go

```diff
@@ -985,3 +985,25 @@ func TestStatusAPIStatements(t *testing.T) {
 		t.Fatalf("expected queries\n\n%v\n\ngot queries\n\n%v", expectedStatements, statementsInResponse)
 	}
 }
+
+func TestStatusAPICommandQueue(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+
+	numNodes := 3
+	testCluster := serverutils.StartTestCluster(t, numNodes, base.TestClusterArgs{})
+	defer testCluster.Stopper().Stop(context.Background())
+
+	// Verify that we can get the command queue without error from any node, and that
+	// the response won't be empty.
```

Comment:
This test still needs a bit of work — seems to pass without the fix. Probably need to increase cluster size and/or hit a different range to repro.